dlhub_sdk/tests/test_dlhub_client.py

```diff
@@ -73,9 +74,9 @@ def test_run(dl):
     assert res.result(timeout=60) == 'Hello world!'
 
 
-@mark.skipif(not is_gha, reason='Avoid running this test except on larger-scale tests of the system')
-@mark.skip
-def test_submit(dl):
+# @mark.skipif(not is_gha, reason='Avoid running this test except on larger-scale tests of the system')
+# @mark.skip
+def test_submit(dl, mocker):
```

Comment:
nice job with mocking

Reply:
Shall we go-ahead and remove the commented-out lines?